### PR TITLE
prov/efa: fix two compilation issues

### PR DESCRIPTION
**Pull request by rwespetal**
Check for 128 bit support and change a strdup call.

### CODE REVIEW

**Comment by rwespetal** [comment]
bot:aws:retest

File: prov/efa/src/rxr/rxr_pkt_type_req.c [hunk 1]
```diff
@@ -2262,6 +2262,7 @@ int rxr_pkt_proc_compare_rta(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 	cmp_data = src_data + rx_entry->total_len;
 	offset = 0;
 
+#ifdef HAVE___INT128
```
Comment (wzamazon) [inline prov/efa/src/rxr/rxr_pkt_type_req.c:2265]: Can you add some comments in the commit message about this change? From the PR title, I know it is to fix a compilation issue. but it is not clear from the the commit message.

**Comment by shefty** [comment]
I don't know what the Intel CI is doing, but whatever it is is unrelated to this PR.

File: prov/efa/src/rxr/rxr_pkt_type_req.c (hunk 1 above)
Comment (rwespetal) [reply prov/efa/src/rxr/rxr_pkt_type_req.c:2265]: Done

**Review by wzamazon — APPROVED** [review]

**Pull request merged**